tools/RunVersionController.ps1

```diff
@@ -233,7 +233,14 @@ function Bump-AzVersion
             Write-Host "Found $currBump version bump for $($localDependency.ModuleName)"
             if ($currBump -eq [PSVersion]::MAJOR)
             {
-                $versionBump = [PSVersion]::MAJOR
+                # if the module is GAed, we don't consider it's a major bump.
+                if($localVersion -eq '1.0.0'){
+                    if($versionBump -ne [PSVersion]::MAJOR){
```

Comment:
Does this if condition need else block if no default value of $versionBump? Otherwise, $versionBump is null.